mne/io/eeglab/eeglab.py

```diff
@@ -443,3 +535,62 @@ def __init__(self, input_fname, events=None, event_id=None, tmin=0,
             reject=reject, flat=flat, reject_tmin=reject_tmin,
             reject_tmax=reject_tmax, add_eeg_ref=False, verbose=verbose)
         logger.info('Ready.')
+
+
+def _read_eeglab_events(eeg, event_id=None, event_id_func=None):
+    """Create events array from EEGLAB structure
+
+    An event array is constructed by looking up events in the
+    event_id, trying to reduce them to their integer part otherwise, and
+    entirely dropping them (with a warning) if this is impossible.
+    Returns a 1x3 array of zeros if no events are found."""
+    if event_id_func is None:
+        event_id_func = _strip_to_integer
+    if event_id is None:
+        event_id = dict()
+
+    types = [event.type for event in eeg.event]
+    latencies = [event.latency for event in eeg.event]
+
+    not_in_event_id = set([x for x in types if x not in event_id])
+    not_purely_numeric = set([x for x in not_in_event_id if not x.isdigit()])
```

Comment:
we should double check if this is a requirement by Python 2.6. Otherwise a generator expression instead of the list comperehension should be generally preferred.